playbooks/update_ravendb.yml

```diff
@@ -1,8 +1,8 @@
 ---
-- hosts: ravendb_nodes
+- hosts: all
   remote_user: root
   roles:
-    - role: "ravendb_node"
+    - role: ravendb.ravendb.ravendb_node
       vars:
           ravendb_state: present
           ravendb_version_minor: 6.2
```

Comment:
Maybe we could make our playbooks use `RAVEN_VERSION` env var and if not set default to 6.2?